### ISSUE TITLE
Keyboard nav (edit: lots of stuff) doesn't work on modqueue

### ISSUE THREAD

**Issue by erikdesjardins** [honestbleeps/Reddit-Enhancement-Suite#2621]
Probably related to #2620, ~~probably the same cause as #2617~~ just slightly different.

https://www.reddit.com/r/RESissues/comments/40mhca/bug_keyboard_nav_no_longer_recognizes_the/
(I can confirm this)

We need to add `subredditAbout` to those shortcuts, or improve our regexes.

Edit: NER doesn't work either, apparently https://www.reddit.com/r/RESissues/comments/40oyow/bug_never_ending_reddit_not_working_on_the/
